css/build.css

```diff
@@ -1,3 +1,7 @@
+html {
+  overflow: hidden;
+}
```

Comment:
We may need to figure out a solution. Because if it also brings back the issue with ghost elements on the page after scrolling, then we can't deploy this.